gems/aws-sdk-s3/lib/aws-sdk-s3/client.rb

```diff
@@ -218,6 +220,11 @@ class Client < Seahorse::Client::Base
     #     Used only in `standard` and adaptive retry modes. Specifies whether to apply
     #     a clock skew correction and retry requests with skewed client clocks.
     #
+    #   @option options [String] :defaults_mode ("legacy")
+    #     TODO: See Aws::ConfigurationDefaults Documentation
```

Comment:
You can link with `{Aws::ConfigurationDefaults}` https://gist.github.com/phansch/db18a595d2f5f1ef16646af72fe1fb0e#linking-objects-